composer.json

```diff
@@ -114,7 +114,6 @@
         "drupal/search_api_mark_outdated": "^1.0",
         "drupal/simple_sitemap": "^3.10",
         "drupal/slick": "^2.6",
-        "drupal/thunder_admin": "^4.0.2",
```

Comment:
We need something in the update docs to download thunder_admin before the update and remove it afterwards. Otherwise it's not possible to uninstall it during the update

Reply:
We already have that